test/benchmarks/benchmark_compare256.cc

```
@@ -90,6 +90,9 @@ BENCHMARK_COMPARE256(avx512, compare256_avx512, test_cpu_features.x86.has_avx512
 #ifdef ARM_NEON
 BENCHMARK_COMPARE256(neon, compare256_neon, test_cpu_features.arm.has_neon);
 #endif
+#ifdef POWER8_VSX
+BENCHMARK_COMPARE256(power8, compare256_power8, test_cpu_features.power.has_arch_2_07);
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If BENCHMARK_DEFINE_F is a macro then please configure it.

(unknownMacro)